GTG/gtk/browser/main_window.py

```diff
@@ -1033,6 +1033,11 @@ def update_start_date(self, widget, new_start_date):
         for task in self.get_pane().get_selection():
             task.date_start = new_start_date
 
+        # Changing the start date of a task may take it out of the current pane
+        # (e.g. setting the start date of an Actionable task to tomorrow)
+        # See #1039
+        self.get_pane().task_filter.changed(Gtk.FilterChange.DIFFERENT)
```

Comment:
This can also be achieved by calling the `refresh()` method of the task pane:
Like so: `self.get_pane().refresh()`

Reply:
Oh nice. Much more readable!